include/dfrn.php

```diff
@@ -0,0 +1,791 @@
+<?php
+require_once('include/items.php');
+require_once('include/Contact.php');
+require_once('include/ostatus.php');
+
+/**
+ * @brief Generates the atom entries for delivery.php
+ *
+ * This function is used whenever content is transmitted via DFRN.
+ *
+ * @param array $items Item elements
+ * @param array $owner Owner record
+ *
+ * @return string DFRN entries
+ */
+function dfrn_entries($items,$owner) {
+
+	$doc = new DOMDocument('1.0', 'utf-8');
+	$doc->formatOutput = true;
+
+	$root = dfrn_add_header($doc, $owner, "dfrn:owner", "", false);
+
+	if(! count($items))
+		return trim($doc->saveXML());
+
+	foreach($items as $item) {
+		$entry = dfrn_entry($doc, "text", $item, $owner, $item["entry:comment-allow"], $item["entry:cid"]);
+		$root->appendChild($entry);
+	}
+
+	return(trim($doc->saveXML()));
+}
+
+/**
+ * @brief Generate an atom feed for the given user
+ *
+ * This function is called when another server is pulling data from the user feed.
+ *
+ * @param App $a
+ * @param string $dfrn_id
+ * @param string $owner_nick Owner nick name
+ * @param string $last_update Date of the last update
+ * @param int $direction
```

Comment:
OK this happens :-D Then add something like `Available values can be -1, 0 or 1`